decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -56,6 +56,8 @@ def reorder(proposals)
             proposals.order(proposal_votes_count: :desc)
           when "recent"
             proposals.order(created_at: :desc)
+          else
+            proposals
```

Comment:
I'd prefer to avoid this branch if we can (it was actually there before but I removed it). I mean, if this is not an expected code path, we should not provide it. I think the better fix is https://github.com/decidim/decidim/issues/1455#issuecomment-306781847 (if it works).

I intentionally not tested this case since I thought the "weird case" could never happen. But I probably should've since it would've catch a bug in my code... :sweat: